src/sysdolphin/baselib/lobj.h

```diff
@@ -46,34 +46,34 @@
 
 typedef struct _HSD_LightPoint {
     f32 cutoff;
-    u8 point_func;
+    u32 point_func;
     f32 ref_br;
     f32 ref_dist;
-    u8 dist_func;
+    u32 dist_func;
```

Comment:
Are these enums, or pointers? Or unsigned constants?

Reply:
Not sure, I don't think their point of use has been decomped yet.

Reply:
I kind of would like there to be a `typedef s32 UnkEnum;` in `types.h` at this point, but what do you think?

Reply:
I don't think we're certain it's an enum, and hiding the type behind an alias seems to add another layer of indirection for little gain.

Reply:
Hm, alright.